problem2.rb

```diff
@@ -0,0 +1,15 @@
+puts "Enter Alphabet to print:"
+alphabet=gets.chomp!
+
+valid_num=false
+until valid_num==true
+  puts "Enter number of copies to create:"
+  num_copies=Integer(gets.chomp!)
+  
+  valid_num=num_copies>=0? true : false
+  puts "Wrong Input! Please Enter Again" unless valid_num
```

Comment:
if you have only string then use single quote.
double quote do two things. 
1. Interpolation
2. Save string.
That's why it is costly as compared to single quote